modules/ROOT/pages/ms-dynamics/ms-service-bus-connector-reference.adoc

```diff
@@ -1049,19 +1043,19 @@ Sends a message to a Topic
 ** AUTO
 ** ALWAYS
 ** NEVER |  Correlation strategy id for operation. |  | x
-| Body a| Any |  The body of the Message |  `#[payload]` | 
-| JMS Type a| String |  The JMSType identifier header of the Message |  | 
-| Correlation ID a| String |  The JMSCorrelationID header of the Message |  | 
-| Send Content Type a| Boolean |  Whether or not the body content type should be sent as a property |  true | 
-| Content Type a| String |  The content type of the message's body |  | 
-| Send Encoding a| Boolean |  Whether or not the body outboundEncoding should be sent as a Message property |  true | 
-| Encoding a| String |  The encoding of the message's body |  | 
-| Reply To a| <<ServiceBusJmsDestination>> |  The destination where a reply to this Message should be sent |  | 
-| User Properties a| Object |  The custom user properties that should be set to this Message |  | 
-| JMSX Properties a| <<JmsxProperties>> |  The JMSX properties that should be set to this Message |  | 
-| Persistent Delivery a| Boolean |  If true, the Message is sent using the PERSISTENT JMSDeliveryMode. |  | 
-| Priority a| Number |  The default JMSPriority value to use when sending the message. |  | 
-| Time To Live a| Number |  Defines the default time the message is in the broker before it expires and is discarded. |  | 
+| Body a| Any |  The body of the Message |  `#[payload]` |
+| JMS Type a| String |  The JMSType identifier header of the Message |  |
+| Correlation ID a| String |  The JMSCorrelationID header of the Message |  |
+| Send Content Type a| Boolean |  Whether or not the body content type should be sent as a property |  true |
+| Content Type a| String |  The content type of the message's body |  |
+| Send Encoding a| Boolean |  Whether or not the body outboundEncoding should be sent as a Message property |  true |
+| Encoding a| String |  The encoding of the message's body |  |
+| Reply To a| <<ServiceBusJmsDestination>> |  The destination where a reply to this Message should be sent |  |
+| User Properties a| Object |  The custom user properties that should be set to this Message |  |
```

Comment:
```suggestion
| User Properties a| Object |  The custom user properties that should be set for this Message |  |
```
??
This and the next couple of fields say "to this message" but I think it should be for?